README

```diff
@@ -0,0 +1,44 @@
+Instructions for building and installing w3 on Unix
```

Comment:
Please remove and transfer this information to the README, if necessary.

I don't think you need to mention modulefiles. Just list what is required for building the library.